### DISCUSSION TITLE
引用参考文献时混用“(author, year)”和“author (year)”

### DISCUSSION THREAD

**Discussion by JSYUs** [zotero-chinese/wiki#165] [求助 / Q&A]
我是一名经济学研究者, 我在一篇paper插入references的时候, 往往同时使用到"(Author, Year)"和"Author (year)"两种形式, 据我所知, Zotero只是支持一种形式的引用输出.
我想问的是, 现在是否已经存在了Zotero插件可以满足输出2种引用格式的. 谢谢.

**Comment by northword** [comment]
对前者，正常添加。对后者，添加时勾选省略作者，这样输出为`(xxxx)`，然后在前面手动输入姓名。

https://zotero-chinese.com/user-guide/ms-word-plugin.html#%E7%9C%81%E7%95%A5%E4%BD%9C%E8%80%85-%E5%9C%A8%E6%96%87%E6%9C%AC%E4%B8%AD%E4%BD%BF%E7%94%A8%E4%BD%9C%E8%80%85

**Reply by JSYUs** [reply]
谢谢你!
